include/open62541/server.h

```diff
@@ -1974,4 +2015,13 @@ _UA_END_DECLS
 #include <open62541/server_pubsub.h>
 #endif
 
+#ifdef UA_ENABLE_ROLE_PERMISSION
+UA_EXPORT UA_StatusCode
+setUserRole_settings(UA_Server* server, UA_String roleName,
```

Comment:
1. I don't understand what these methods are doing just from their naming.
    So other users won't either.
3. Use the UA_ prefix for all public API.
4. Roles are identified by a NodeId, not by a string.